src/Platform/Microsoft.Testing.Platform/Resources/PlatformResources.resx

```diff
@@ -489,4 +489,14 @@ Read more about Microsoft Testing Platform telemetry: https://aka.ms/testingplat
   <data name="PlatformCommandLineIgnoreExitCodeOptionDescription" xml:space="preserve">
     <value>Do not report non successful exit value for specific exit codes (e.g. '--ignore-exit-code 8;9' ignore exit code 8 and 9 and will return 0 in these case)</value>
   </data>
-</root>
+  <data name="PlatformCommandLineExitOnProcessExit" xml:space="preserve">
```

Comment:
Please use the same pattern as for the other entries `PlatformCommandLineExitOnProcessExitOptionDescription`